internals/core_enums.py

```diff
@@ -175,43 +200,50 @@
         (STAGE_DEP_DEPRECATION_TRIAL, GATE_ORIGIN_TRIAL),
         (STAGE_DEP_EXTEND_DEPRECATION_TRIAL, GATE_EXTEND_ORIGIN_TRIAL),
         (STAGE_DEP_SHIPPING, GATE_SHIP),
-        (STAGE_DEP_REMOVE_CODE, None)]
+        (STAGE_DEP_REMOVE_CODE, None)],
+    FEATURE_TYPE_ENTERPRISE_ID: [
+        (STAGE_ENT_ROLLOUT, None)]
   }
 
 # Prototype stage types for every feature type.
 STAGE_TYPES_PROTOTYPE: dict[int, Optional[int]] = {
     FEATURE_TYPE_INCUBATE_ID: STAGE_BLINK_PROTOTYPE,
     FEATURE_TYPE_EXISTING_ID: STAGE_FAST_PROTOTYPE,
     FEATURE_TYPE_CODE_CHANGE_ID: None,
-    FEATURE_TYPE_DEPRECATION_ID: None
+    FEATURE_TYPE_DEPRECATION_ID: None,
+    FEATURE_TYPE_ENTERPRISE_ID: None
```

Comment:
Let's use final trailing commas on all these list items.  That will make future diffs easier to read.
https://google.github.io/styleguide/jsguide.html#features-arrays-trailing-comma